draft-wang-ppm-dap-taskprov.md

```diff
@@ -156,14 +156,14 @@ struct {
 
     /* This determines the query type for batch selection and the
     properties that all batches for this task must have. */
-    QueryConfig query_config;
+    opaque query_config<0..2^16-1>;
```

Comment:
Should minimum length be `1` to enforce it is present?

Reply:
Done.

Note that eventually we'll set these to be the actual minimum length, which is 15 in this case (the smallest possible QueryConfig).